### PR TITLE
Add joi schema validation

### PR DESCRIPTION
**Pull request by FlorentinDUBOIS**

### CODE REVIEW

**Comment by miton18** [comment]
Need conflict resolution

File: src/models/env.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  PROXY_JWT_ISSUER: Joi.string().required(),
+  PROXY_JWT_AUDIENCE: Joi.string().required(),
+  PROXY_MONGODB_ADDON_URI: Joi.string().required(),
+  PROXY_MONGODB_ADDON_USER: Joi.string(),
+  PROXY_MONGODB_ADDON_PASSWORD: Joi.string(),
+  PROXY_MONGODB_ADDON_DB: Joi.string(),
+  PROXY_MONGODB_ADDON_HOST: Joi.string(),
+  PROXY_MONGODB_ADDON_PORT: Joi.number(),
+  PROXY_PEPPER: Joi.string().required()
+}).or('PROXY_MONGODB_ADDON_URI', [
```
Comment (miton18) [inline src/models/env.js:20]: You can add Warp10 env var ;-)

**Review by miton18 — CHANGES_REQUESTED** [review]

File: src/models/env.js [hunk 2]
```diff
@@ -14,15 +14,16 @@ const EnvSchema = Joi.object().keys({
   PROXY_MONGODB_ADDON_DB: Joi.string(),
   PROXY_MONGODB_ADDON_HOST: Joi.string(),
   PROXY_MONGODB_ADDON_PORT: Joi.number(),
-  PROXY_PEPPER: Joi.string().required()
+  PROXY_PEPPER: Joi.string().required(),
+  PROXY_WARP10_WRITE_TOKEN: Joi.string(),
+  PROXY_WARP10_URI: Joi.string()
```
Comment (miton18) [inline src/models/env.js:19]: 12 Env vars in readme and 14 in Joi model, one of us is lying ;-)

**Review by miton18 — CHANGES_REQUESTED** [review]

File: src/models/env.js (hunk 2 above)
Comment (FlorentinDUBOIS) [reply src/models/env.js:19]: The cake is a lie !

**Pull request merged**